src/styles/index.scss

```diff
@@ -2,2 +1,90 @@
+* {
+  padding: 0;
   margin: 0;
+  box-sizing: border-box;
+}
+
+$size: 100px;
+$border: 1px solid black;
+$background: #eee;
+$text-color: #000;
+$text-size: 30px;
+$gap: 1px;
+$hover-color: #ffbfcb;
+$transition-duration: 0.5s;
+$width-calendar: 7 * $size + $gap * 6;
+$days: (
+  sun: 6,
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+);
+
+.block_calendar {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
The BEM naming convention suggests that block elements should be named using a single underscore to separate the block from the element. For example, `.block__calendar` instead of `.block_calendar`.